README.md

```diff
@@ -37,30 +35,41 @@ In `/etc/hosts` add this:
 ```
 
 ### Setup Database
+- Follow the steps [here](https://mariadb.com/kb/en/installing-and-using-mariadb-via-docker/) to install mariadb via docker.
+- Then run the docker container using the following command.
+```bash
+docker run --detach --name mariadb-tumlive --env MARIADB_USER=root --env MARIADB_PASSWORD=root --env MARIADB_ROOT_PASSWORD=example --restart always -p 3306:3306 mariadb:latest`
+```
+- Alternatively, install mariadb on its own.
+- Create the database `tumlive` using [this]([tum-live-starter.zip](https://github.com/joschahenningsen/TUM-Live/files/8505487/tum-live-starter.zip)) script.
+- Optional: Use [JetBrains DataGrip](https://www.jetbrains.com/datagrip/) to open this project and then run the script there to automatically set up a demo database.
 
-- get mariadb from your favourite package manager or docker (I recommend this option)
-- `docker run --name mariadb -e MYSQL_ROOT_PASSWORD=example -p 3306:3306 -d mariadb:latest`
-- create the database `tumlive`
-
-### Get go running locally
+### Installing go
 
-- Install go 1.18
-- Preferably use Jetbrains GoLand and open this project
-- Edit Configuration > Environment 
-  - Add environment variables from `variables-backend.example.env`.
-- Start the app
-- Head over to localhost:8081
-- Happy coding! :sparkles:
+- Install **go 1.18** by following the steps [here](https://go.dev/doc/install)
+- Preferably use [JetBrains GoLand](https://www.jetbrains.com/go/) and open this project as it simplifies this entire process
+- Run `npm i` in the `./web` directory to install the required node modules
+- Run `go get ./...` to install the required go modules
+- If you want to customize the environment variables (for example mariadb username and password), copy the `config.yaml` file over to your home directory
```

Comment:
```suggestion
- If you want to customize the configuration (for example mariadb username and password), copy the `config.yaml` file over to `$HOME/.TUM-Live/config.yaml` and make your changes there to prevent accidentally committing them.
```